src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
Fair enough... I'd forgotten that the rest of the source files get concatenated after this one. Yes, let's leave it as is.